www/less/nativeSectionHeaderPlaceholder.less

```diff
@@ -15,19 +18,39 @@ To easily debug and see if the header and the html are in sync:
 A good article to test is "enwiki > Beach Boys" because it has lots of long section titles.
 */
 
-.section_heading {
+
+/* Styling for section headers which will have native overlays */
+H1.section_heading, H2.section_heading, H3.section_heading {
```

Comment:
Now we'll need to scope the placeholder styling (which we need to as closely match the native overlay as possible) to just h1, h2 and h3.